### DISCUSSION TITLE
Mouse permanently unresponsive

### DISCUSSION THREAD

**Discussion by alextoff** [LizardByte/community#58] [Sunshine] [answered]
Hi, I've just installed Sunshine, and I'm using the Moonlight client on an NVIDIA Shield.   Everything was working perfectly for about 10 minutes until my mouse froze.   Now it's just unresponsive.  I disconnected from the host, restarted the host PC and server, restarted my NVIDIA Shield, but still no luck getting it working.   However, when I left or right click I see "Warming: Button state of mouse_button [1] (or [3]) does not match the desired state," so it's still detecting mouse clicks, but something's bugged so that it's not actually doing anything.  The cursor doesn't move, and the mouse clicks don't do anything.  Any idea what the issue could be?  

UPDATE:  After several reboots of both my PC and Shield, and swapping the mouse between USB ports it's working again.  Not sure what happened and why it was staying that way though.

**Comment by RussellAult** [comment, marked as answer]
I'm assuming the host is running Windows. If so, do you have the Task Manager running, or perhaps some software open that has administrator privileges?

I was recently reminded that Windows can get _very_ picky about how different pieces of software interact with each other, particularly for software with elevated privileges (or other system-level software). Normally this is a very good thing, since it's an easy way to stop viruses from, for example, taking control over the Task Manager, but in remote desktop situations like Sunshine it has a side-effect: if the currently-active window is one of these security-conscious applications Windows will simply refuse to acknowledge any input from remote-access software. This is particularly annoying if you accidentally open the Task Manager while using Sunshine, since the computer will immediately become "unresponsive" and you'll have to find another way to close the Task Manager (RDP _will_ let you interact with elevated windows, but it of course comes with its own set of problems when trying to use Sunshine).

**Reply by elyckir** [reply]
Turning off User Account Control has no effect.
Are there still no workarounds?

**Reply by igeryu** [reply]
Probably not a good idea, long-term, but I can further corroborate this by saying that when I ran Sunshine, on Windows, with Administrator rights, I was able to continue using my remote keyboard and mouse.

**Reply by ReenigneArcher** [reply]
Are you running Sunshine via the installed service?

**Reply by ConteXCrown** [reply]
Make sure Sunshine.exe is run by System and not by User (check in Taskmanager). Make sure you use the sunshine installer and not the sunshine portable. The portable doesnt run as System automatically.

**Comment by alextoff** [comment]
Definitely sounds like what's happening.  The first time it happened I believe was when I right clicked Parsec's system tray icon which must have woken it up.  Also freezes if Parsec is launched, even when not connected, and when opening task manager.  I've been able to restart Sunshine whenever it happens and all is good from then on.  It's possible something was running in the background that first instance where a full restart didn't fix it.  In any case, thanks for clearing it up, it makes sense.

**Comment by ghost** [comment]
I just got a similar issue, i was trying out Sunshine and ran Moonlight-QT on the same machine, had it start Steam Big Screen and since the gamepad didnt work, i had it install its "xbox gamepad driver", it then immediately restarted and now NO mouse OR keyboard works in windows at all!

Works fine in BIOS and in Startup menus etc. but once windows loads, power is turned off to both devices and will never function again, and when i try unplug and back in, the leds blink once and then no more. Not ieven any of the SafeModes helps... 

Now this could be Steams fault, ie that "driver" but its more likely that whatever Sunshine does to "hide" kb&mouse is still in effect since the computer rebooted without properly stopping it.

Im really im a bind here...

**Reply by ReenigneArcher** [reply]
Sunshine doesn't do anything to your physical devices
